### PR TITLE
#35 add support for wildfly 14

### PR DESCRIPTION
**Pull request by adietish**
Signed-off-by: Andre Dietisheim <adietish@redhat.com>

### CODE REVIEW

File: server-wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/servertype/capabilities/ExtendedServerPropertiesAdapterFactory.java [hunk 1]
```diff
@@ -61,6 +61,8 @@ public ServerExtendedProperties getExtendedProperties(IServer s) {
 				return new Wildfly120ExtendedProperties(s);
 			if( SERVER_WILDFLY_130.equals(typeId) )
 				return new Wildfly130ExtendedProperties(s);
+			if( SERVER_WILDFLY_140.equals(typeId) )
+				return new Wildfly130ExtendedProperties(s);
```
Comment (robstryker) [inline server-wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/servertype/capabilities/ExtendedServerPropertiesAdapterFactory.java]: Added a new class, did not use it. Fix.

File: server-wildfly/src/test/java/org/jboss/tools/rsp/server/wildfly/test/beans/MockServerCreationUtilities.java [hunk 2]
```diff
@@ -50,6 +49,7 @@
 	private static final String wildfly_11_0_jar = "wf11.0.0.mf.jboss-as-server.jar";
 	private static final String wildfly_12_0_jar = "wf12.0.0.mf.jboss-as-server.jar";
 	private static final String wildfly_13_0_jar = "wf13.0.0.mf.jboss-as-server.jar";
+	private static final String wildfly_14_0_jar = "wf14.0.0.mf.jboss-as-server.jar";
```
Comment (robstryker) [inline server-wildfly/src/test/java/org/jboss/tools/rsp/server/wildfly/test/beans/MockServerCreationUtilities.java]: Did you actually add this file? I can't tell by looking at the PR.

Comment (adietish) [reply server-wildfly/src/test/java/org/jboss/tools/rsp/server/wildfly/test/beans/MockServerCreationUtilities.java]: added a comment as requested (wfl13 and wfl14 jars are missing and we should get rid of the jars at some point)

**Comment by adietish** [comment]
@robstryker corrected, pls have a look

**Pull request merged**